styles.css

```
@@ -0,0 +1,78 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-image:
+    linear-gradient(
+      115deg,
+      rgba(58, 58, 158, 0.8),
+      rgba(136, 136, 206, 0.7)
+    ),
+    url("https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg");
+  width: 100%;
+  height: auto;
+  background-repeat: no-repeat;
+  font-size: 1.2vw;
+  color: #fff;
+}
+
+#head {
```

```suggestion
.head {
```

Please use dot instead of hashtag. dot is more recommended to use in css while hashtag is mostly used in javascript

---

**[OPTIONAL]**
 
- Hey, you've done really great. this is just more like an extra, I think it would be better to say `classes` in place of the `dot[.]` and `id's` in place of the `hashes[#]` as this is more understood as related to web development in general. :+1: